webapp-django/crashstats/signature/views.py

```diff
@@ -556,7 +556,11 @@ def signature_summary(request, params):
 
                 graphics.append(entry)
 
-        facets['adapter_vendor_id'] = graphics
+        facets['adapter_vendor_id'] = sorted(
```

Comment:
Add a comment that explains *why* this couldn't be done in the query and why it has to be done in django.